src/NuGet/Microsoft.CodeAnalysis.Compilers.nuspec

```diff
@@ -2,9 +2,13 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
   <metadata minClientVersion="3.3">
     <id>Microsoft.CodeAnalysis.Compilers</id>
-    <summary>.NET Compiler Platform ("Roslyn")</summary>
+    <summary>
+      .NET Compiler Platform ("Roslyn").
```

Comment:
Should this say it's the compiler layer? This otherwise sounds like it's the whole thing, which it's not.